docs/sphinx/source/examples/colpali-document-retrieval-vision-language-models-cloud.ipynb

```diff
@@ -14,6 +14,8 @@
                 "\n",
                 "# Vespa 🤝 ColPali: Efficient Document Retrieval with Vision Language Models\n",
                 "\n",
+                "For a simpler example of using ColPali, where we use one Vespa document = One PDF page, see [simplified-retrieval-with-colpali-vlm_Vespa-cloud](simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb).\n",
```

Comment:
```suggestion
                "For a simpler example of using ColPali, where we use one Vespa document = One PDF page, see [simplified-retrieval-with-colpali-vlm_Vespa-cloud](https://pyvespa.readthedocs.io/en/latest/examples/simplified-retrieval-with-colpali-vlm_Vespa-cloud.html).\n",
```

Reply:
Doesn't sphinx auto solve this?

Reply:
I think it does if you link the html, but with the `.ipynb`-suffix, I think you get the raw notebook - eg. [https://pyvespa.readthedocs.io/en/latest/examples/simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb](https://pyvespa.readthedocs.io/en/latest/examples/simplified-retrieval-with-colpali-vlm_Vespa-cloud.ipynb)

Reply:
Thanks